pom.xml

```diff
@@ -47,6 +47,7 @@
 
         <spring.springframework.version>5.0.5.RELEASE</spring.springframework.version>
         <spring.data.version>2.0.1.RELEASE</spring.data.version>
+        <spring.boot.version>2.0.0.RELEASE</spring.boot.version>
```

Comment:
Why not use more recent release of spring boot version?

Reply:
do we need to keep align with newest version ?

Reply:
No,  because sometimes there are vulnerability fixes, e.g., https://spring.io/blog/2018/06/14/spring-project-vulnerability-reports-published

Reply:
OK. will update it.